README.md

```diff
@@ -276,7 +276,63 @@ export default createCable({
 
 ### Testing
 
-_⏳ Coming soon_
+For testing your channel you can use test cable implementation from `@anycable/core/testing`.
+
+By using test cable implementation you can test channel's output actions. All actions store in cable `outgoing` property.
+Also test implementation helps to test channel `disconnect` event.
+
+For example we have the following channel implementation.
+
+```js
+import { Channel } from "@anycable/core";
+
+class ChatChannel extends Channel {
+  static identifier = "ChatChannel";
+
+  async speak(message) {
+    return this.perform("speak", { message });
+  }
+
+  async leave() {
+    // some custom logic
+    return this.disconnect();
+  }
+}
+```
+
+We can test it like this (using `Jest`):
+
+```js
+import { Channel } from './channel.js'
+import { TestCable } from '@anycable/core/testing'
+
+describe('ChatChannel', () => {
+  let channel: Channel
+  let cable: TestCable
+
+  beforeEach(() => {
+    cable = new TestCable()
+    channel = new Channel()
+    cable.subscribe(channel)
+  })
+
+  it('perform an speak action', async () => {
+    await channel.perform('speak', { message: 'hello' })
```

Comment:
Let's provide an example with the channel's own API:

```suggestion
    await channel.speak('hello')
```

Reply:
Yeah, my bad. Copied code to readme from two sources and forgot to change it